Core.DomainModel/Shared/LifeCycleEventType.cs

```diff
@@ -4,6 +4,6 @@ public enum LifeCycleEventType
     {
         Created = 0,
         Updated = 1,
-        Deleted = 2
+        Deleting = 2
```

Comment:
check if this results in issues on the api

Reply:
no issues. It's correct in the context where it is used. The tracking db has a different enum and the api as well